src/_includes/accessibility/accessibility-manual.html

```diff
@@ -93,19 +93,18 @@ <h4 class="margin-top-xl text-muted">If an icon represents an interactive elemen
 
   <div class="margin-bottom-lg margin-top-lg">
 {% highlight html %}
-<a href="#navigation-main" title="Skip to main navigation">
+<a href="#navigation-main" aria-label="Skip to main navigation">
   <i class="fa fa-bars" aria-hidden="true"></i>
 </a>
 {% endhighlight %}
     <small class="text-muted">an icon being used as a link to a navigation menu</small>
 
 {% highlight html %}
-<a class="btn btn-danger" href="path/to/settings" title="Delete">
-  <i class="fa fa-trash-o" aria-hidden="true"></i>
+<a class="btn btn-danger" href="path/to/settings" aria-label="Delete">
+  <i class="fa fa-trash-o" aria-hidden="true" title="Delete"></i>
```

Comment:
as per the comment in the thread from @karlgroves we should be careful about encouraging the use of the title attribute in case people don't understand its limitations (perhaps encouraging them to use a custom tooltip for this purpose) for accessibility reasons but strictly speaking the code you're proposing isn't incorrect